rsolr.gemspec

```diff
@@ -27,13 +27,14 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {spec}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
-  
+
   s.required_ruby_version      = '>= 1.9.3'
-  
+
   s.requirements << 'Apache Solr'
 
   s.add_dependency 'builder', '>= 2.1.2'
   s.add_dependency 'faraday', '>= 0.9.0'
```

Comment:
FWIW it looks like the faraday dependency will never be able to <1 due to faraday-net_http's requirements.  Should this be bumped up to `>= 1.0` to not give false hopes?

Reply:
I don't think that's true -- `faraday-net_http` is only a dependency of faraday depending on what version of faraday you are using. If you are using faraday < 1, you don't have `faraday-net_http` at all. I think there's no reason for rsolr to stop letting users use a version of faraday they may be currently using without problems -- especially on a minor version release.

Reply:
@jrochkind `faraday-net_http` is being required here on the next line.  From what I can see in rubygems, `faraday-net_http` has only every supported `faraday` 1.0 and higher.  By adding a hard dependency on `faraday-net_http` here then rsolr would essentially be requiring faraday 1.0+.

Reply:
I think we could have done this differently to preserve backwards compat with all previously supported faraday versions -- we didn't need to explicitly depend on `faraday-net_http`, and didn't need to do the `require` if Faraday was less than 2, which we could have checked for. 

I would have appreciated y'all giving this more time for discussion when we are in the middle of a discussion -- what's the rush/urgency?